.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Please update `.gitignore` to ignore `.idea/`

Reply:
dammit I thought I had handled that ... will update shortly

Reply:
oh it was in the ocm PR :-)

Reply:
ok now in builder PR as well